docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx

```diff
@@ -44,6 +44,8 @@ or for the purposes of this walkthrough, you can click the link below all these
 
 <img src={useBaseUrl("/img/tutorial/tutorial_03a_database_connection_string_link.png" )} width="600" />{" "} <br/><br/>
 
+Please note, if you are trying to connect to another locally running database such as Clickhouse (whether on host or another container), and you get the message `The port is closed.`, then you need to adjust the HOST to `host.docker.internal`
```

Comment:
Is there something that's likely Clickhouse-specific here, or would it be best to remove the "such as Clickhouse" to keep it generic?

Reply:
I think this part of the docs is meant for supporting UX more than admin/docker folks.

Feels like this note belongs somewhere between the "Connecting to Databases" and "Using Docker/Docker-compose" section